scoresheets/StoringGroceries.tex

```diff
@@ -1,24 +1,31 @@
-\begin{scorelist}[timelimit=5]
+\begin{scorelist}[timelimit=6]
 	\scoreheading{Main Goal}
 	\scoreitem{15}{Navigating to the table}
 	\scoreitem[5]{15}{Perceiving object and categorizing it correctly}
 	\scoreitem[5]{50}{Picking up an object for transportation to the cabinet}
 	\scoreitem[5]{15}{Perceiving objects in shelf and saying on which layer the currently handled object should be placed}
 	\scoreitem[5]{15}{Placing an object in the cabinet}
 	\scoreitem[5]{50}{Placing an object next to similar objects on the cabinet}
+	\scoreitem{300}{Pouring cereal into the container}
 
 	\scoreheading{Bonus Rewards}
 	\scoreitem{200}{Opening the cabinet door without human help}
+	\scoreitem{50}{Picking up an object from the shopping bag}
```

Comment:
I'd suggest to also give 70 points here since it is more difficult. It is comparable to picking up the tiny/heavy objects.

Reply:
I would change the overall scores later. For now this is just to get the different score items into the rulebook